Sources/MeiliSearch/Client.swift

```diff
@@ -212,17 +212,85 @@ public struct MeiliSearch {
   // MARK: Keys
 
   /**
-   Each instance of MeiliSearch has three keys: a master, a private, and a public. Each key has a given
-   set of permissions on the API routes.
+   Get all keys.
 
-   - parameter masterKey:  Master key to access the `keys` function.
    - parameter completion: The completion closure used to notify when the server
    completes the query request, it returns a `Result` object that contains `Key` value.
    If the request was sucessful or `Error` if a failure occured.
    */
-  public func keys(
+  public func getKeys(
+    _ completion: @escaping (Result<Results<Key>, Swift.Error>) -> Void) {
+    self.keys.getAll(completion)
+  }
+
+  /**
+   Get one key's information using the key value.
+
+   - parameter key:  The key value.
+   - parameter completion: The completion closure used to notify when the server
+   completes the query request, it returns a `Result` object that contains `Key` value.
+   If the request was sucessful or `Error` if a failure occured.
+   */
+  public func getKey(
+    key: String,
+    _ completion: @escaping (Result<Key, Swift.Error>) -> Void) {
+    self.keys.get(key: key, completion)
+  }
+
+  /**
+    Create an API key.
+
+   - parameter description:  Description of the key.
+   - parameter actions:  Actions permitted by the key.
+   - parameter indexes:  Indexes on which the key has permissions.
+   - parameter expiresAt:  Date in ISO-8601 format when the key is expired.
```

Comment:
`- parameter expiresAt:  A future date in ISO-8601 format when the key should be considered invalid.`
Or something like this 🎃

Reply:
Is this oke for you: `A future date in ISO-8601 format when the key becomes invalid.` ?